docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -988,31 +1007,40 @@ thor: []
       </sect3>
 
       <sect3 id="FileAccessCosts">
-        <title>File Access Cost</title>
+        <title>File Access Costs</title>
```

Comment:
The title should be something like "File Cost".  File Access Costs doesn't include Storage At Rest cost.

Also, the paragraph on line 1003-1007, is no longer correct. The two types of storage costs has been separated into 2 different fields and can now be viewed as separate fields.